src/wallet/rpc/addresses.cpp

```diff
@@ -541,6 +541,10 @@ RPCHelpMan getaddressinfo()
                         {
                             {RPCResult::Type::STR, "label name", "Label name (defaults to \"\")."},
                         }},
+                        {RPCResult::Type::ARR, "use_txids", "",
+                        {
+                            {RPCResult::Type::STR_HEX, "txid", "The ids of transactions involving this wallet which received with the address"},
```

Comment:
94b20a9f9e81e5b0ef45a0259b81be83b602c325

- `s/which/that/` ("which" would require a comma-space preceding it) 
- maybe add "have already" and `s/the/this/` 

```suggestion
                            {RPCResult::Type::STR_HEX, "txid", "The ids of transactions involving this wallet that have already received with this address"},
```

Reply:
I think the current string is better. Not sure why it would require a comma.